hippod/hasher.py

```diff
@@ -57,17 +57,22 @@ def check_sum_object_issue(o):
             msg = "object data currupt - version missing: {}".format(str(o))
             raise ApiError(msg)
 
-        # Use a sorted dictionary
-        buf = __sum_dict(o)
-        return hashlib.sha1(buf.encode('utf-8')).hexdigest()
+        #majorbuf = str.join("", o["title"] + o["categories"])
+        cat_buf = ''.join(map(str, o["categories"]))
+        majorbuf = o["title"].replace(" ", "") + cat_buf
```

Comment:
why replace the spaces by NOPs? Not required, right ... Mhh, Not sure is the vanilla string is sufficiently